web-components/src/components/cards/ProjectCard/ProjectCard.ImageChildren.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, useTheme } from '@mui/material';
+
+import UserInfoCard, { Party } from '../../../components/user/UserInfoCard';
+
+type Props = {
+  program?: Party;
+};
+
+export const ProgramImageChildren = ({ program }: Props) => {
+  const theme = useTheme();
+
+  return (
+    <>
+      {program && (
+        <Box
+          sx={{
+            position: 'absolute',
+            bottom: theme.spacing(3.75),
+            left: theme.spacing(5),
```

Comment:
Any reason for this not being 5,5 ? Seems slightly off to me but maybe this is as intended.

Reply:
cc @erikalogie

Reply:
I know we have a few new spacing issues so just want to double check before we merge.

Reply:
![Screenshot from 2023-08-14 10-34-29](https://github.com/regen-network/regen-web/assets/12519942/60df05fd-dd94-424a-87db-aea388145583)